packages/open-payments/src/client/grant.test.ts

```diff
@@ -10,23 +23,98 @@ describe('grant', (): void => {
     openApi = await createOpenAPI(config.OPEN_PAYMENTS_AS_OPEN_API_URL)
   })
 
-  const axiosInstance = defaultAxiosInstance
-  const logger = silentLogger
-  const client = 'https://example.com/.well-known/pay'
+  const deps = {
+    axiosInstance: defaultAxiosInstance,
+    logger: silentLogger,
+    client: 'https://example.com/.well-known/pay'
+  }
+
+  describe('routes', () => {
+    const url = 'http://localhost:1000'
+    const accessToken = 'someAccessToken'
+
+    describe('request', () => {
+      test('calls post method with correct validator', async (): Promise<void> => {
+        const mockResponseValidator = ({ path, method }) =>
+          path === '/' && method === HttpMethod.POST
+
+        jest
+          .spyOn(openApi, 'createResponseValidator')
+          .mockImplementation(mockResponseValidator as never)
+
+        const postSpy = jest.spyOn(requestors, 'post').mockResolvedValueOnce({})
```

Comment:
Do we need the `mockResolvedValueOnce` here if we have the mocked `post: jest.fn()` above?

Reply:
Yup, don't need it. Thanks